### ISSUE TITLE
Add rate limiting per day

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-server#17261]
Add rate limiting per day for users.

**Comment by AlexandraMoga** [comment]
@willdurand I've tested the add-on submission rate limiting on stage using the submission API and DevHub and these are my results:

- user restrictions:
  - 3 submission requests/minute rule - i.e. 3 submissions (201), 4th submission denied (429)
  - 20 submission requests/hour rule - i.e. 20 submissions (201), 21st submission denied (429)
  - 24 submission requests/day rule - seems to be working, but I've added an observation below
 
From what I observed, the rate limit will take into consideration every submission request, successful or not; so, for example, I've submitted 20 add-on versions and for the 21 version I reached the 20 submissions/hour restriction. At this point I have 21 submission requests, with only 20 successful uploads. Once the 1 hour rate limit expires, I can successfully upload 3 more versions, the next submission attempts being denied as the daily rate limit kicks in. This means that I end up only with 23 successful uploads per day, not 24. 

As a side note, I did change IPs once in a while between submissions, to make sure I don't hit an IP restriction instead of a user restriction.
